conf/fam.yaml.template

```diff
@@ -36,6 +36,7 @@ FAM:
     auth_source_ldap_attr_login: uid
     auth_source_ldap_groups_base: cn=groups,cn=accounts,dc=example,dc=com
     external_usergroup_name: "admins"
+    default_auth_source_ldap_account_password: changeme
```

Comment:
Lets give some different password than one used here.

Reply:
This is just a template which needs to be copied and change the values. This need not to be identical with CI at all.

Reply:
sure, suggest one? 
(not that it really matters, it's an example)

Reply:
changed to `password`